content/en/service_management/service_level_objectives/guide/slo_types_comparison.md

```diff
@@ -0,0 +1,45 @@
+| **SLO alerting ([Error Budget][1] or [Burn Rate][2] Alerts)**         | Available for all metric-based SLOs                                                                                                       | Available for SLOs based on Metric Monitor types only (not available for Synthetic Monitors or Service Checks)                                                      | Not available for Time Slice SLOs                                                    |
+| [**SLO Status Corrections**][3]                                       | Correction periods are ignored from SLO status and error budget calculations                                                              | Correction periods are ignored from SLO status and error budget calculations                                                                                        | Correction periods are counted as uptime in SLO status and error budget calculations |
+| [**SLO Widgets**][4]                                                  | Available for Metric-based SLOs                                                                                                           | Available for all Monitor-based SLO types                                                                                                                           | Available for Time Slice SLOs                                                        |
+| [**SLO Data Source**][5]                                              | Available for this SLO type (up to 15 months of historical data available)                                                                | Not available for this SLO type                                                                                                                                     | Not yet available for Time Slice SLOs                                                |
+| **Handling missing data in the SLO calculation**                      | Missing data is ignored in SLO status and error budget calculations                                                                       | Missing data is handled based on the [underlying Monitor's configuration][6]                                                                                        | Missing data is treated as uptime in SLO status and error budget calculations        |
+| **Uptime Calculations**                                               | Uptime calculations are based on the underlying Monitor<br><br>If groups are present, overall uptime requires *all* groups to have uptime | Uptime is calculated by looking at discrete time chunks, not rolling time windows<br><br>If groups are present, overall uptime requires *all* groups to have uptime | N/A                                                                                  |
+
```

Comment:
For the uptime calculation info for time slice SLOs, let's link to the section on Uptime calculations https://docs-staging.datadoghq.com/esther/docs-6808-time-slice-slo/service_management/service_level_objectives/time_slice/#uptime-calculations